clang/include/clang/Sema/Sema.h

```diff
@@ -2445,6 +2445,8 @@ class Sema final : public SemaBase {
   ///
   /// \param FD The FieldDecl to apply the attribute to
   /// \param E The count expression on the attribute
+  /// \param NestedTypeLevel The pointer indirection level where the attribute
```

Comment:
Hmm... why does this have to be a certain number deep, and not just calculated/stored on that type?